src/app/modules/main/chat_section/module.nim

```diff
@@ -282,31 +282,11 @@ method chatContentDidLoad*(self: Module) =
 method setActiveItemSubItem*(self: Module, itemId: string, subItemId: string) =
   self.controller.setActiveItemSubItem(itemId, subItemId)
 
-method makeChatWithIdActive*(self: Module, chatId: string) =
```

Comment:
@borismelnik I am wondering why did you remove this method? This is very important method for us. Please revert it.